episodes/introduction_to_anonymization.md

```diff
@@ -0,0 +1,345 @@
+(0028, 0100) Bits Allocated                      US: 16
+(0028, 0101) Bits Stored                         US: 12
+(0028, 0102) High Bit                            US: 11
+(0028, 0103) Pixel Representation                US: 0
+(0028, 1050) Window Center                       DS: [50, 50]
+(0028, 1051) Window Width                        DS: [350, 350]
+(0028, 1052) Rescale Intercept                   DS: '-1024.0'
+(0028, 1053) Rescale Slope                       DS: '1.0'
+(0032, 1033) Requesting Service                  LO: 'CHIPSOFT'
+(0040, 1001) Requested Procedure ID              SH: 'CT5001IV'
+(0054, 1001) Units                               CS: 'HU'
+(00e1, 0010) Private Creator                     LO: 'ELSCINT1'
+(00e1, 1036) Private tag data                    CS: 'YES'
+(00e1, 1040) [Image Label]                       SH: 'Cor IMR med'
+(00e1, 1046) Private tag data                    OB: Array of 512 elements
+(01f1, 0010) Private Creator                     LO: 'ELSCINT1'
+(01f1, 1001) [Acquisition Type]                  CS: 'SPIRAL'
+(01f1, 1002) [Unknown]                           CS: 'STANDARD'
+(01f1, 100e) [Unknown]                           FL: 0.0
+(01f1, 1027) [Rotation Time]                     DS: '0.33'
+(01f1, 1032) [Image View Convention]             CS: 'RIGHT_ON_LEFT'
+(01f1, 104a) [Unknown]                           SH: 'DOM'
+(01f1, 104b) [Unknown]                           SH: '128x0.625'
+(01f1, 104d) [Unknown]                           SH: 'NO'
+(01f1, 104e) [Unknown]                           SH: 'Chest'
+(01f1, 1054) Private tag data                    IS: '11'
+(01f1, 1056) Private tag data                    LO: '30.0451206729581'
+(01f7, 0010) Private Creator                     LO: 'ELSCINT1'
+(01f7, 1022) [Unknown]                           UI: 1.3.46.670589.33.1.63849049343237673200010.5507538603167078985
+(07a1, 0010) Private Creator                     LO: 'ELSCINT1'
+(07a1, 1010) [Tamar Software Version]            LO: '4.0.0'
+(7fe0, 0010) Pixel Data                          OB: Array of 309328 elements
+```
+
+
+:::::::::::::::::::::::::::::::::::::::  challenge
+
+## What metadata is safe
+
+Can you identify for this CT which metadata is probably safe i.e. does not lead to patient identification.
+When might you keep such data?
+
+:::::::::::::::  solution
+
+## Solution
+
+Metadata about the machine, image type and file type are pretty safe. 
+This data may be very useful is we are sorting through a lot of DICOMS 
+and only want a certain type of image, or need to create tabular data which helps harmonization.
+
+:::::::::::::::::::::::::
+
+::::::::::::::::::::::::::::::::::::::::::::::::::
+
+
+We can modify elements of our dicom metadata
+
+```python
+elem = ds[0x0010, 0x0010]
+elem.value
+```
+```output
+
+```
+```python
+elem.value = 'Citizen^Almoni'
+elem
+```
+In some cases, if we are dealing with a standard element, we can modify elements by keyword:
```

Comment:
I think that `PatientName` is an attribute, or a property of the class. I'd use the specific correct term here

Reply:
Done